src/main/java/uk/gov/ons/ctp/response/casesvc/message/CaseCreationReceiver.java

```diff
@@ -1,13 +1,31 @@
 package uk.gov.ons.ctp.response.casesvc.message;
 
+import com.godaddy.logging.Logger;
+import com.godaddy.logging.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.integration.annotation.MessageEndpoint;
+import org.springframework.integration.annotation.ServiceActivator;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
 import uk.gov.ons.ctp.response.casesvc.message.sampleunitnotification.SampleUnitParent;
+import uk.gov.ons.ctp.response.casesvc.service.CaseService;
+
+/** Receive a new case from the Collection Exercise service. */
+@MessageEndpoint
+public class CaseCreationReceiver {
+  private static final Logger log = LoggerFactory.getLogger(CaseCreationReceiver.class);
+
+  @Autowired private CaseService caseService;
 
-/** The reader of SampleUnitParents from queue */
-public interface CaseCreationReceiver {
   /**
    * To process SampleUnitParents read from queue
    *
    * @param caseCreation the java representation of the message body
    */
-  void acceptSampleUnit(SampleUnitParent caseCreation);
+  @Transactional(propagation = Propagation.REQUIRED, readOnly = false, value = "transactionManager")
```

Comment:
I have no idea why we need to specify these values explicitly. `@Transactional` should be plenty